### ISSUE TITLE
Flash1: Graphic showing how to create accessible name in Flash has no accessible name

### ISSUE THREAD

**Issue by dd8** [w3c/wcag#1139]
Figure 1 in technique Flash 1 has this code:

```
<figure id="figure-flash-authoring-accessibility-panel">
 <img src="img/acc_name_scrn.jpg">			       
 <figcaption>Figure 1The Accessibility panel in the Flash authoring environment.</figcaption>
</figure>
```

https://www.w3.org/WAI/WCAG21/Techniques/flash/FLASH1.html

Chrome and Firefox a11y inspectors show graphic as having null name. VoiceOver with Safari reads "acc underscore name underscore scrn dot jpg, image"

**Comment by dd8** [comment]
The same is also true of Flash2, Flash21, Flash28, Flash29, Flash32, Flash5, Flash6, PDF1, PDF10, PDF11, PDF12, PDF17, PDF20, PDF4, PDF6, PDF9, SL20, SL28, SL8.

Looks like these all assume figcaption provides an accessible name

**Comment by alastc** [comment]
I think the Flash ones will be deprecated soon, but the PDF ones are likely to stick around.

**Comment by dd8** [comment]
The problem seems worse than I first reported, most of the `img` elements in the WCAG 2.1 techniques don't have `alt` so don't have an accessible name, but the equivalent WCAG 2.0 technique do have `alt` on `img`

For example:

https://www.w3.org/TR/WCAG20-TECHS/SCR32.html (has alt)
https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR32 (no alt)

https://www.w3.org/TR/WCAG20-TECHS/G167.html (has alt)
https://www.w3.org/WAI/WCAG21/Techniques/general/G167 (no alt)

https://www.w3.org/TR/WCAG20-TECHS/F69.html (has alt)
https://www.w3.org/WAI/WCAG21/Techniques/failures/F69 (no alt)

From searching the source on Github it looks like there's no `alt` in the majority of places where `img` is used.

**Comment by fstrr** [comment]
Closing due to the removal of Flash techniques

**Comment by dd8** [comment]
I think this should be re-opened - it's not just about the Flash techniques - it's most/all of the WCAG 2.1 techniques with `<img>` elements

https://github.com/w3c/wcag/issues/1139#issuecomment-663854412

**Comment by fstrr** [comment]
I don't think WCAG 2.1 is being updated any more—note that the [WCAG 2.1 version of SCR32](https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR32) is missing alt text, the [WCAG 2.2 version of SCR32](https://www.w3.org/WAI/WCAG22/Techniques/client-side-script/SCR32) has fixed that issue.

I can create a new issue for the eight images in WCAG 2.2 that don't have `alt` text (assuming this regex is correct: `<img(?:(?!\balt=)[^>])*>`.
